include/franka_lightweight_interface/franka_lwi_communication_protocol.h

```diff
@@ -77,16 +101,11 @@ struct StateMessage {
 
 template<std::size_t DOF>
 struct CommandMessage {
+  ControlType controlType = NONE;
+  Joints<DOF> jointPosition;
+  Joints<DOF> jointVelocity;
   Joints<DOF> jointTorque;
+  EEPose eePose;
+  EETwist eeTwist;
+  EETwist eeWrench;
 };
-
-// --- Conversion helpers --- //
-inline std::array<datatype, 3> vec3DToArray(const Vec3D& vec3D) {
-  return std::array<datatype, 3>({vec3D.x, vec3D.y, vec3D.z});
-}
-
-inline std::array<datatype, 4> quaternionToArray(const Quaternion& quaternion) {
-  return std::array<datatype, 4>({quaternion.w, quaternion.x, quaternion.y, quaternion.z});
-}
-
-}
```

Comment:
Would be nice to keep that last bracket here :smile: My whole citrified is in the proto namespace otherwise...

Reply:
wow you're so picky, fine fine